console/src/main/java/org/nervos/ckb/RpcRequest.java

```diff
@@ -52,7 +53,7 @@ public static NodeInfo localNodeInfo() throws IOException {
     }
 
     public static List<CellOutputWithOutPoint> getCellsByTypeHash(String typeHash, long fromBlockNumber, long toBlockNumber) throws IOException {
-        return ckbService.getCellsByTypeHash(typeHash, fromBlockNumber, toBlockNumber).send().getCells();
+        return ckbService.getCellsByLockHash(typeHash, fromBlockNumber, toBlockNumber).send().getCells();
```

Comment:
Now that we change it to `LockHash`, the param name should be updated as well.

Reply:
It's a mistake and I will update it.